### PR TITLE
TileView: add Vue and React code snippets

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: concepts/05 Widgets/TileView/05 Customize Tile Appearance.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 For a minor customization of tiles, you can define [specific fields](/api-reference/10%20UI%20Widgets/dxTileView/1%20Configuration/items '/Documentation/ApiReference/UI_Widgets/dxTileView/Configuration/items/') in item data objects. For example, the following code generates three tiles: the first is disabled, the second is not customized, the third is hidden.
 
 ---
-#####jQuery
+##### jQuery
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/TileView/05 Customize Tile Appearance.md:4]: Please make the same changes that were made to a similar file for the [Tabs](https://github.com/DevExpress/devextreme-documentation/pull/329/files#diff-73a7b01724c5a539d83ef67c507f494fR3) widget.

File: concepts/05 Widgets/TileView/15 Keyboard Support.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ You can implement a custom handler for a key using the [registerKeyHandler(key,
 
     <!--JavaScript-->
     function registerKeyHandlers () {
-        let tileView =  $("#tileViewContainer").dxTileView("instance");
+        const tileView =  $("#tileViewContainer").dxTileView("instance");
```
Comment (RomanTsukanov) [inline concepts/05 Widgets/TileView/15 Keyboard Support.md]: This code example has been put into an include. You can replace this entire paragraph with a single line:

```
#include common-code-register-key-handler
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: concepts/05 Widgets/TileView/15 Keyboard Support.md (hunk 2 above)
Comment (EugeniyKiyashko) [reply concepts/05 Widgets/TileView/15 Keyboard Support.md]: fixed

File: concepts/05 Widgets/TileView/05 Customize Tile Appearance.md (hunk 1 above)
Comment (ksercs) [reply concepts/05 Widgets/TileView/05 Customize Tile Appearance.md:4]: fixed

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**